packages/web/src/pages/upload-page/store/utils/sagaHelpers.ts

```diff
@@ -37,5 +39,15 @@ export function* reportSuccessAndFailureEvents({
     })
   )
 
-  yield* all([...successEvents, ...failureEvents].map((e) => put(e)))
+  const rejectedEvents = range(numRejected).map((i) =>
```

Comment:
Not sure I follow this comment, but I think yes.

We have two types of metrics:
- metrics that capture which page of the upload flow someone is on
- per track, what happened to the upload (success/failure/reject)
It gets tricky & complicated with playlist upload (what if one track gets rejected?)

In this change, I make is so that the "rejected" counts towards getting to the completion page of the upload flow so we can take # users on complete / # users on start sign up and have a better proxy of success.

I really think this whole thing needs a rewrite though

Reply:
ah yeah that makes sense, that rejected still counts for completion page